proparse/src/main/java/org/prorefactor/proparse/ParserSupport.java

```diff
@@ -92,90 +88,53 @@ void addInnerScope() {
     currentScope = new SymbolScope(session, currentScope);
   }
 
-  public boolean isInDynamicNew() {
-    return inDynamicNew;
-  }
-
-  public void setInDynamicNew(boolean flag) {
-    inDynamicNew = flag;
-  }
-
-  /** Mark a node as "operator" */
-  static void attrOp(JPNode ast) {
-    ast.attrSet(IConstants.OPERATOR, IConstants.TRUE);
-  }
+  // Functions triggered from proparse.g
 
+  /**
+   * When parsing class, all methods are added as part of the classState rule.
+   * TODO Don't do that if RCodeUnit object is there
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1135)